misc/graylog.conf

```diff
@@ -689,3 +689,27 @@ proxied_requests_thread_pool_size = 32
 # Setting this to an empty value, leaves it up to system libraries and the used JDK to chose a default.
 # Default: TLSv1.2,TLSv1.3  (might be automatically adjusted to protocols supported by the JDK)
 #enabled_tls_protocols= TLSv1.2,TLSv1.3
+
+# Enable prometheus exporter HTTP server.
+# Default: false
+#prometheus_exporter_enabled = false
+
+# IP address and port for the prometheus exporter HTTP server.
+# Default: 127.0.0.1:9832
+#prometheus_exporter_bind_address = 127.0.0.1:9832
+
+# Path to the prometheus exporter core mapping file. If this option is enabled, the full built-in core mapping is
+# replaced with the mappings in this file.
+# This file is monitored for changes and updates will be applied at runtime.
+# Default: ./prometheus-exporter-mapping-core.yml
+#prometheus_exporter_mapping_file_path_core = prometheus-exporter-mapping-core.yml
```

Comment:
Should the core file mappings really be configurable? As far as I can see, there is only benefit in allowing the custom optional mappings to be configured.

Perhaps storing the core mappings as a compiled resource is enough. What do you think?

Reply:
@danotorrey The idea was, that it's possible to change the core mapping without any code changes if really needed. (e.g. a bug fix)

This option is not something that is supposed to be used very often. We might want to mention that in the comment for the option?

Reply:
As discussed, keeping the overridable core mappings makes sense.